### PR TITLE
Installation doc improvements

### PR DESCRIPTION
**Pull request by redcatbear**
- Removed duplicate lines
- Added links to Python releases

**Things To Remember:** 

1. Be sure to merge into the right branch. The choices are:
* **`master`** if your PR runs on the current version of Tableau.
* **`dev-2020.1`**
* **`2020.2`**
2. You should update at least one of the following:
* The changelog of the project you modified.
* The root `README.md` if the PR is a release of the SDK, Connector Packager, or TDVT.  
=====

### CODE REVIEW

**Review by pvanderknyff — APPROVED** [review]
Looks good to me, getting reviews from some others before merging

**Review by rosswbrown — APPROVED** [review]

**Comment by pvanderknyff** [comment]
Merged, thanks @redcatbear!

**Pull request merged**
